src/community/pom.xml

```diff
@@ -16,6 +16,9 @@
   <artifactId>community</artifactId>
   <packaging>pom</packaging>
   <name>Community Space</name>
+  <modules>
+    <module>ogcapi/web-ogcapi</module>
+  </modules>
```

Comment:
This is the issue... IntelliJ trying to be helpful and adding the new module in the wrong pom...